R/CohortComparisonDiagnostics.R

```diff
@@ -69,7 +69,7 @@ computeCohortOverlap <- function(connectionDetails = NULL,
   
   sql <- SqlRender::loadRenderTranslateSql(
     "CohortOverlap.sql",
-    packageName = "CohortDiagnostics",
+    packageName= utils::packageName(),
```

Comment:
This somehow seems less safe, because we're no long explicit about which package contains the resource. What is the reason for this change?

Reply:
Trusting the R namespacing is cleaner to me; it also prevents copy paste errors when people make study packages. I would even prefer a usage of a constant to propagating the reuse of the same string throughout projects.